backend/server/routes/realtime.ts

```diff
@@ -1,8 +1,11 @@
-const feathers = require('feathers');
+import * as feathers from 'feathers';
+import * as socketio from 'feathers-socketio';
+
+import UserService from '../services/user_service';
+
 const realtime = feathers();
-const socketio = require('feathers-socketio');
 
-realtime.configure(socketio({ wsEngine: 'uws' }, (io) => {
```

Comment:
Yea, it caused me some issues with docker since I had to rebuild the c++ for the correct operating system (added a couple seconds to our build). If we dont need that dependency anymore, that would be nice to speed up build times :)